petfinder-data/data_cleaner.py

```diff
@@ -77,13 +78,12 @@ def find_org(org_id, org_name) -> pd.DataFrame:
 
     return org_dict_df
 
-
 if __name__ == "__main__":
 
     # read in the raw data for animals
-    data_file = DATA_FOLDER / "chicago_animals.pkl"
+    data_file = DATA_FOLDER / f"{City_State.replace(', ', '_').lower()}_animals.pkl"
```

Comment:
Since we have a bit of logic that requires a city and state, comma delimited, to be passed in, we might consider putting in some assertion statements earlier up to make sure that users pass in correctly formatted values for City_State, and fail right away (instead of waiting until it gets to here before throwing errors).

Maybe the assertion could be that there's exactly one comma in the string, and that it doesn't start or end with a comma? Or, if we run `City_State.split(', ')` we could assert that the length of the result is 2?

Reply:
Added assertion to make sure location is correct